### PR TITLE
Provide a way to override brick cell content after the updateContent

### PR DESCRIPTION
**Pull request by wfsttam**
- provides a means of showing placeholder content

### CODE REVIEW

File: Source/Cells/BrickCell.swift [hunk 1]
```diff
@@ -24,6 +24,16 @@ public protocol BrickCellTapDelegate: UIGestureRecognizerDelegate {
     func didTapBrickCell(brickCell: BrickCell)
 }
 
+public protocol OverrideContentSource: class {
+    func overrideContent(forBrickCell brickCell: BrickCell)
```
Comment (rubencagnie) [inline Source/Cells/BrickCell.swift]: What about the `resetCell` function to clean up the placeholder views?

Comment (rubencagnie) [inline Source/Cells/BrickCell.swift]: Let's do it "swift 3"-style
```
func overrideContent(for cell: BrickCell)
```

Comment (wfsttam) [inline Source/Cells/BrickCell.swift]: fixed

Comment (wfsttam) [inline Source/Cells/BrickCell.swift]: fixed

File: Source/Cells/BrickCell.swift [hunk 2]
```diff
@@ -24,6 +24,20 @@ public protocol BrickCellTapDelegate: UIGestureRecognizerDelegate {
     func didTapBrickCell(brickCell: BrickCell)
 }
 
+public protocol OverrideContentSource: class {
+    func overrideContent(for brickCell: BrickCell)
+    func resetContent(for brickCell: BrickCell)
+}
+
+extension OverrideContentSource {
+    public func overrideContent(for brickCell: BrickCell) {
```
Comment (rubencagnie) [inline Source/Cells/BrickCell.swift]: Why is this optional?

Comment (wfsttam) [inline Source/Cells/BrickCell.swift]: fixed

**Comment by wfsttam** [comment]
Fixes #46

**Pull request merged**